src/instructlab/configuration.py

```diff
@@ -359,40 +404,24 @@ def get_default_config() -> Config:
             ),
         ),
         train=_train(
-            train_args=TrainingArgs(
-                model_path=DEFAULTS.MODEL_REPO,
-                data_path=DEFAULTS.DATASETS_DIR,
-                ckpt_output_dir=DEFAULTS.CHECKPOINTS_DIR,
-                data_output_dir=DEFAULTS.INTERNAL_DIR,
-                max_seq_len=4096,
-                max_batch_len=10000,
-                num_epochs=10,
-                effective_batch_size=3840,
-                save_samples=250000,
-                learning_rate=2e-6,
-                warmup_steps=800,
-                is_padding_free=False,
-                random_seed=42,
-                lora=LoraOptions(
-                    rank=4,
-                    alpha=32,
-                    dropout=0.1,
-                    target_modules=["q_proj", "k_proj", "v_proj", "o_proj"],
-                ),
-                deepspeed_options=DeepSpeedOptions(
-                    cpu_offload_optimizer=False,
-                    cpu_offload_optimizer_ratio=1,
-                    cpu_offload_optimizer_pin_memory=False,
-                    save_samples=None,
-                ),
-            ),
-            torch_args=TorchrunArgs(
-                node_rank=0,
-                nnodes=1,
-                nproc_per_node=1,
-                rdzv_id=123,
-                rdzv_endpoint="127.0.0.1:12222",
-            ),
+            model_path=DEFAULTS.MODEL_REPO,
+            data_path=DEFAULTS.DATASETS_DIR,
+            ckpt_output_dir=DEFAULTS.CHECKPOINTS_DIR,
+            data_output_dir=DEFAULTS.INTERNAL_DIR,
+            max_seq_len=4096,
+            max_batch_len=10000,
+            num_epochs=10,
+            effective_batch_size=3840,
+            save_samples=250000,
+            lora_quantize_dtype="nf4",
+            lora_rank=4,
+            lora_alpha=32,
+            lora_dropout=0.1,
+            lora_target_modules=["q_proj", "k_proj", "v_proj", "o_proj"],
+            deepspeed_cpu_offload_optimizer=False,
+            deepspeed_cpu_offload_optimizer_ratio=1,
+            deepspeed_cpu_offload_optimizer_pin_memory=False,
```

Comment:
- optimizer_ratio & pin_memory can move to expert args
- lora dropout, alpha, and target modules can move 
- lora_quantize_dtype should be moved but not in this PR since it requires a bigger lift. Issue for it here: https://github.com/instructlab/instructlab/issues/1673